bigquery_etl/cli/query.py

```diff
@@ -117,7 +118,41 @@ def query(ctx):
     default=False,
     is_flag=True,
 )
-def create(name, sql_dir, project_id, owner, init):
+@click.option(
+    "--dag",
+    "-dag",
```

Comment:
Single-dash options should be a single letter.
```suggestion
    "-d",
```